components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/utils/MappingUtil.java

```diff
@@ -543,9 +549,20 @@ public static int mapSecuritySchemeListToInt(List<String> securityScheme) {
                 default:break;
             }
         }
+        
+        if (securityScheme.size() == 0) {
+            securitySchemeValue = 3;
```

Comment:
Mmhhh going forward if someone do not provide authentication scheme defualt will be fall to OAuth better to fix it at the first place